src/persist-client/src/internal/state.rs

```diff
@@ -1107,6 +1107,19 @@ pub enum HollowBlobRef<'a, T> {
     Rollup(&'a HollowRollup),
 }
 
+/// A rollup that is currently being computed.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Arbitrary, Serialize)]
+pub struct ActiveRollup {
+    /// 0 if no rollup is being computed.
+    pub start_ms: u64,
```

Comment:
I'd much rather store an `Option<ActiveRollup>` instead of special-casing the zero value here.

Partly for general data-modelling reasons - makes it harder to forget to handle the "no rollup in progress" case - but also because I think it will round-trip with existing states a little better.